game/fx/common_effects.go

```diff
@@ -0,0 +1,42 @@
+package fx
+
+import (
+	"duel-masters/game/cnd"
+	"duel-masters/game/match"
+)
+
+func GiveTapAbilityToAllies(card *match.Card, ctx *match.Context, alliesFilter func(card *match.Card) bool, tapAbility func(card *match.Card, ctx *match.Context)) {
+	// This is added for the case where the card is added to the field. There is another creature
+	// that doesn't initially have a tap abbility but should receive one. The change doesn't propagate fast
+	// enough to the FE and that creature doesn't get tap ability until another action takes places.
+	// This is an ugly workaround.
+	FindFilter(
```

Comment:
This part is slightly ugly, unsure what's the better way to do this if there is one.